### ISSUE TITLE
Rstudio gives an error after opening

### ISSUE THREAD

**Issue by pk12344** [rstudio/rstudio#12892]
R Studio gives the following error when I opened it: 

### RStudio Version

RStudio 2023.03.0+386 "Cherry Blossom " (3c53477a, 2023-03-09) for macOS

Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) RStudio/2023.03.0+386 Chrome/108.0.5359.179 Electron/22.0.3 Safari/537.36

### Error message

[No error available]

### Process Output

The R session exited with code 1. 

Error output:

```
[No errors emitted]
```

Standard output:

```
[No output emitted]
```

### Logs

```
2023-03-22T13:11:20.957046Z [rsession-] ERROR system error 92 (Illegal byte sequence) [str: Bełej, len: 6, from: UTF-8, to: ISO8859-1]; OCCURRED AT core::Error rstudio::r::util::(anonymous namespace)::iconvstrImpl(const std::string &, const std::string &, const std::string &, bool, std::string *) src/cpp/r/RUtil.cpp:187; LOGGED FROM: rstudio::core::Error rstudio::session::modules::spelling::(anonymous namespace)::checkSpelling(const json::JsonRpcRequest &, json::JsonRpcResponse *) src/cpp/session/modules/SessionSpelling.cpp:225
2023-03-22T14:03:54.828093Z [rsession-] ERROR system error 92 (Illegal byte sequence) [str: Bełej, len: 6, from: UTF-8, to: ISO8859-1]; OCCURRED AT core::Error rstudio::r::util::(anonymous namespace)::iconvstrImpl(const std::string &, const std::string &, const std::string &, bool, std::string *) src/cpp/r/RUtil.cpp:187; LOGGED FROM: rstudio::core::Error rstudio::session::modules::spelling::(anonymous namespace)::checkSpelling(const json::JsonRpcRequest &, json::JsonRpcResponse *) src/cpp/session/modules/SessionSpelling.cpp:225
2023-03-22T14:03:54.828093Z [rsession-] ERROR system error 92 (Illegal byte sequence) [str: Bełej, len: 6, from: UTF-8, to: ISO8859-1]; OCCURRED AT core::Error rstudio::r::util::(anonymous namespace)::iconvstrImpl(const std::string &, const std::string &, const std::string &, bool, std::string *) src/cpp/r/RUtil.cpp:187; LOGGED FROM: rstudio::core::Error rstudio::session::modules::spelling::(anonymous namespace)::checkSpelling(const json::JsonRpcRequest &, json::JsonRpcResponse *) src/cpp/session/modules/SessionSpelling.cpp:225

```
             

I already tried to install R and Rstudio again, but it doesn't work. I use the newest MacOs. Can someone help me?  Thanks!

**Comment by hahahaL** [comment]
version:
R:3.6 Rstudio:2022.02.2+485 (Prairie Trillium)
meet the same issue ,the mes like these:
```ERROR system error 84 (Invalid or incomplete multibyte or wide character) [str: 一, len: 9, from: UTF-8, to: ISO8859-1]; OCCURRED AT rstudio::core::Error rstudio::r::util::{anonymous}::iconvstrImpl(const string&, const string&, const string&, bool, std::string*) src/cpp/r/RUtil.cpp:187; LOGGED FROM: rstudio::core::Error rstudio::session::modules::spelling::{anonymous}::checkSpelling(const rstudio::core::json::JsonRpcRequest&, rstudio::core::json::JsonRpcResponse*) src/cpp/session/modules/SessionSpelling.cpp:225```

**Comment by ronblum** [comment]
@pk12344 Thank you for raising the issue! RStudio Desktop 2023.03 is only supported on MacOS 11 and above. The most recent version available for MacOS 10.15 is [2022.07.2+576.pro12](https://dailies.rstudio.com/version/2022.07.2+576.pro12/).

@hahahaL This looks like a different issue. Can you please [open a new issue](https://github.com/rstudio/rstudio/issues/new?assignees=&labels=bug%2Cnew&template=1_bug_report.md&title=) for it?

**Comment by ronblum** [comment]
Closing this since this is an unsupported configuration, but please let me know if earlier releases, such as 2022.07, don't work. We might not be able to address it since this is an older MacOS environment, but it would at least be helpful for other users to know.
